grammar.js

```diff
@@ -1425,7 +1465,9 @@ module.exports = grammar({
     param: $ => {
       const extern_name = field('extern_name', $.identifier)
       const name = field('name', choice(
-        $.identifier, $.instance_var, $.class_var, $.macro_var, $.macro_expression,
+        $.identifier, alias($.identifier_method_call, $.identifier),
+        $.instance_var, $.class_var,
+        $.macro_var, $.macro_expression,
```

Comment:
Nitpick for all these changes: personally I would prefer to see the arguments all on the same line, or all on separate lines.

Reply:
This isn't currently enforced, but I just found the eslint rule for it, might add that myself.

Reply:
Please do!